main.gms

```diff
@@ -27,7 +27,7 @@ $setGlobal fReadCountryDB 'yes'
 $setGlobal fReadCountryCalib 'yes'
 
 $setGlobal fHorizon '2010*2100'
-$setGlobal fEndY 2020
+$setGlobal fEndY 2018
```

Comment:
maybe change this back to 2020

Reply:
Done https://github.com/giannou/OPEN-PROM/commit/29c9a16825f3454ce49c45f61ec1c1b6540fa33f